assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -30,7 +30,7 @@ import { isValidUrl, withProtocol } from '../../../../utils/url';
 import useElementsWithLinks from '../../../../utils/useElementsWithLinks';
 import { LinkInput, Row } from '../../../form';
 import { SimplePanel } from '../../panel';
-import { ExpandedTextInput } from '../../shared';
+import { Input } from '../../../../../design-system';
 
 const Error = styled.span`
```

Comment:
I think we should update this to use a `<Text />` component. Then we won't have to use old colors and a font size